README.md

```diff
@@ -2,31 +2,162 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Learn how to write a file using the `FileOutputStream` class.
+- Learn how to read a file using the `FileInputStream` class.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Another way to read and write files in Java is through the use of streams. We
+have learned about streams previously when it comes to collections and arrays.
+In this lesson, we'll learn about how we can apply those same concepts to
+working with files.
 
-## Topic 1
+## FileOutputStream
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The `FileOutputStream` class provides an `OutputStream` for writing data out to
+a file. We can construct a `FileOutputStream` by passing in a `File` object or
+a `String` object with the file name.
 
-## Topic 2
+```java
+import java.io.File;
+import java.io.FileOutputStream;
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+public class FileIOMain {
+    public static void main(String[] args) {
+        File file = new File("simple.txt");
 
-## Conclusion
+        FileOutputStream fileOutputStream = new FileOutputStream(file);
+    }
+}
+```
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+Now that we have our `FileOutputStream` instance instantiated, we can use this
+to write out data to a file! It should be noted that `FileOutputStream` is meant
+for writing streams of raw bytes out to a file. The Java Oracle documents
+suggest using this class for writing out imaging data rather than streams of
+characters. Let's look at why this might be the case:
+
+```java
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class FileIOMain {
+    public static void main(String[] args) {
+        File file = new File("simple.txt");
+        String output = "example of writing to a file";
```

Comment:
Not very important, but your example outputs have a period at the end of the String